src/stories/Colours.stories.tsx

```diff
@@ -1,6 +1,50 @@
 import React from 'react';
 import { ColorPalette, ColorItem } from '@storybook/addon-docs/blocks';
 
+import colourTokens from 'build/js/colours';
+import { toCssVariable } from 'utils/change-case';
+
+type TokensProps = {
+  [key: string]: string;
+};
+
+// TODO: Ideally we would import tokens as a prop but type declarations
+// are not implicit at the node level i.e. `tokenKeys.map((key) =>`
+export const ColourVariables = ({ tokens }: TokensProps) => {
+  const tokenKeys = Object.keys(colourTokens) as Array<keyof typeof colourTokens>;
+
+  return (
+    <table className="sb-table sb-table--swatches">
+      <thead>
+        <tr>
+          <th scope="col">Token name</th>
+          <th scope="col">CSS variable</th>
+          <th scope="col">Swatch and value</th>
+        </tr>
+      </thead>
+      <tbody>
+        {tokenKeys.map((key) => {
+          const value = colourTokens[key];
+          const cssValue = /rgb/i.test(key) ? `rgb(${value})` : value;
```

Comment:
This could fail if someone used the letters `rgb` in their variable name. For example, `FlagColourGB`. Doing a regex against the value might be safer than doing it against the key.

Reply:
This line has been removed. Ultimately if we add RGB values we can handle that using a Style Dictionary transform.